IntelFsp2WrapperPkg/Library/FspWrapperMultiPhaseProcessLib/PeiFspWrapperMultiPhaseProcessLib.c

```diff
@@ -352,6 +356,15 @@ FspWrapperMultiPhaseHandler (
     FspMultiPhaseParams.MultiPhaseParamPtr = NULL;
     Status                                 = CallFspMultiPhaseEntry (&FspMultiPhaseParams, FspHobListPtr, ComponentIndex);
 
+    //
+    // Refresh FspHobList pointer stored in HOB.

```

Comment:
When the platforms only support VariableRequest, we will need similar code to refresh FspHobList in FspWrapperVariableRequestHandler() too. (to refresh the pointer after FspHobList point migrated to permanent memory)

Reply:
Done. Thanks for pointing this out.